src/core/tracking.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-underscore-dangle */
+
+import config from 'config';
+import log from 'core/logger';
+
+
+export class Tracking {
+
+  constructor(opts = {}) {
+    this.id = opts.trackingId;
+    this.enabled = opts.enabled;
+    this.logPrefix = `[GA: ${this.enabled ? 'ON' : 'OFF'}]`;
+    this.initialized = false;
+  }
+
+  _ga(args) {
+    if (!this.initialized) {
+      throw new Error('Must call init() first');
+    } else {
+      // eslint-disable-next-line no-undef
+      ga.apply(window, [...args]);
+    }
+  }
+
+  init() {
+    /* istanbul ignore next */
+    if (this.enabled === true) {
+      log.info('Tracking init (Analytics is ON)');
```

Comment:
What do you mean by console binding? The line numbers? I don't think we'd care about them much since this is pretty explicit. I was thinking something like:

```js
log(...args) {
  log.info(this.logPrefix, ...args);
}